distribution/changelog.txt

```diff
@@ -11,6 +11,7 @@
 - Change: [#24069] [Plugin] Plugins are now available in the scenario editor and track designer.
 - Change: [#24135] Compress Emscripten js/wasm files.
 - Change: [#24235] Small changes to RCT1 theme.
+- Change: [#24308] “Software” and “Software (hardware display)” renderers have been merged into a single “Software” renderer.
```

Comment:
Have they? As far as I can see, we just dropped the former and no merging took place.

Reply:
People aren't really aware that the two are more or less the same, the difference is only in how we copy the pixels to something that renders it onto the screen, this will see some changes in the future also.